ScoutSuite/providers/azure/provider.py

```diff
@@ -56,7 +56,7 @@ def get_report_name(self):
         Returns the name of the report using the provider's configuration
         """
         try:
-            return f'azure-tenant-{self.credentials.get_tenant_id()}'
+            return f'azure-tenant-{self.credentials.get_tenant_id}'
```

Comment:
Isn't `get_tenant_id` a method? Isn't it missing `()`s?

Reply:
This is correct, they are added back

Reply:
Looks like only in https://github.com/nccgroup/ScoutSuite/blob/e3850327067b729baf3a3b4cac4b604d571439ae/ScoutSuite/providers/azure/provider.py#L40, not https://github.com/nccgroup/ScoutSuite/blob/e3850327067b729baf3a3b4cac4b604d571439ae/ScoutSuite/providers/azure/provider.py#L59?

Reply:
I mean, you are correct. They were added back in a later commit, this code is outdated.